js/ui/context_menu/ui.menu_base.js

```diff
@@ -132,7 +132,7 @@ class MenuBase extends HierarchicalCollectionWidget {
         return ITEM_CLASS;
     }
 
-    _setAriaSelected() {}
+    _setAriaSelectionAttribute() { return noop; }
```

Comment:
We should not return function, we need to return void. Thus, there should be:
_setAriaSelectionAttribute = noop;